.travis.yml

```diff
@@ -11,7 +11,9 @@ rvm:
   - 1.9.2
   - 1.9.3
   - 2.0.0
-  - 2.1.0
+  - 2.1.7
```

Comment:
I'd be tempted to go with 2.1.9. I don't expect any differences, but it's what puppet-agent ships with currently.